dataplane-policy.yaml

```
@@ -0,0 +1,165 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  annotations:
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+  name: metrics-forwarder-config
+  namespace: openshift-acm-policies
+spec:
+  disabled: false
+  policy-templates:
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+            name: metrics-forwarder-config
+        spec:
+            remediationAction: enforce
+            evaluationInterval:
+                compliant: 2h
+                noncompliant: 45s
+            object-templates:
+                - complianceType: musthave
```

```suggestion
                - complianceType: mustonlyhave
```
This is related to a bug, more details in https://issues.redhat.com/browse/OSD-15177